ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,39 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    self.storage[self.current] = item
 
+    # Update current to next value
+    if self.current < self.capacity - 1:
```

Comment:
Would this reset it back to one an element early?
I think you would be okay with just checking 
`if self.current < self.capacity:`